src/main/java/com/gw2auth/oauth2/server/service/Gw2ApiPermission.java

```diff
@@ -9,33 +9,33 @@
 
 public enum Gw2ApiPermission {
 
-    ACCOUNT("account", 0),
-    BUILDS("builds", 1),
-    CHARACTERS("characters", 2),
-    GUILDS("guilds", 3),
-    INVENTORIES("inventories", 4),
-    PROGRESSION("progression", 5),
-    PVP("pvp", 6),
-    TRADINGPOST("tradingpost", 7),
-    UNLOCKS("unlocks", 8),
-    WALLET("wallet", 9);
+    ACCOUNT("account", 0, OAuth2Scopes.GW2_ACCOUNT),
+    BUILDS("builds", 1, OAuth2Scopes.GW2_BUILDS),
+    CHARACTERS("characters", 2, OAuth2Scopes.GW2_CHARACTERS),
+    GUILDS("guilds", 3, OAuth2Scopes.GW2_GUILDS),
+    INVENTORIES("inventories", 4, OAuth2Scopes.GW2_INVENTORIES),
+    PROGRESSION("progression", 5, OAuth2Scopes.GW2_PROGRESSION),
+    PVP("pvp", 6, OAuth2Scopes.GW2_PVP),
+    TRADINGPOST("tradingpost", 7, OAuth2Scopes.GW2_TRADINGPOST),
+    UNLOCKS("unlocks", 8, OAuth2Scopes.GW2_UNLOCKS),
+    WALLET("wallet", 9, OAuth2Scopes.GW2_WALLET);
 
     private static final List<Gw2ApiPermission> VALUES = List.of(values());
```

Comment:
There is no reason to keep this around since all remaining usages can be replaced with `ALL`.